### ISSUE TITLE
BOTW: Korok Removal Help?

### ISSUE THREAD

**Issue by MonfGeiger** [marcrobledo/savegame-editors#464]
TL;DR I updated CEMU, made sure BOTW was up to snuff, grabbed Second Wind v1.9.4-alpha-6, and after making sure the eventide fix was enabled and everything compiled, i played up to the point where i need to fight Sooga after defrosting him........bit of a problem: the fix didnt remove the korok that was in its place (I guess the mod expected me to NOT have the korok seeds in certain areas.......whoops?)

I do not know how to "remove" this korok, and i kinda dont want to zero my save and start over fresh if i dont have to, so i gather if i can "remove" this korok, and boot it back up, the mod should spawn Sooga where he should be, so.....what would i have to edit in the master editor to maybe get rid of this korok without breaking the game, or would at this point, my only option (besides checking the SW discord) be to start fresh?
![Capture](https://github.com/user-attachments/assets/42b24cb4-45ed-44cd-b598-e16e9437df1c)
![Capture2](https://github.com/user-attachments/assets/81aeff77-cec7-4659-b04d-e847a5010487)
